dev/PipsPager/InteractionTests/PipsPagerTests.cs

```diff
@@ -315,5 +316,51 @@ public void KeyboardPageSelectTest()
                 VerifySelectedPageIndex(2);
             }
         }
+        [TestMethod]
+        [TestProperty("TestSuite", "F")]
+        public void PipSizeWithDifferentOrientationsTest()
+        {
+            using (var setup = new TestSetupHelper("PipsPager Tests"))
+            {
+                Button getButtonSizesButton = FindElement.ByName<Button>("GetPipsPagersButtonSizes");
```

Comment:
why aren't you using the elements object to get this button

Reply:
@StephenLPeters I should've followed the pattern - I fixed that now. But I still don't understand - why are the tests failing on that button invoke?:)

Reply:
agreed I'm not sure yet